openslides/core/models.py

```diff
@@ -243,3 +273,61 @@ class Meta:
     def get_session_store_class(cls):
         from .session_backend import SessionStore
         return SessionStore
+
+
+class Message(RESTModelMixin, models.Model):
```

Comment:
You can not use `prune_elements` for a motion with the flag `stable: true`. Or if you have more unstable slides on a projector than a motion you can also not use the `prune_elements` viewpoint because you would disable other slides as well. So it is no special thing the you have to look for the uuid of the element that you want to disable. If we transform the message and the countdown to a semi model (saved in model but controlled by ProjectorViewSet) we have to care about a lot of additional code and about data redundancy (what if a countdown in model has a different uuid than in projector, what if one is missing, ...). I think it is eaysier to run a short lodash function in the client to find our uuid. (We do this already in current master, I think; not for multiple projectors of course.)